Datadog.Trace.sln

```diff
@@ -1615,6 +1617,16 @@ Global
 		{662B587F-97B5-4CEF-ABF9-6C76A6DBD29E}.Release|x64.Build.0 = Release|x64
 		{662B587F-97B5-4CEF-ABF9-6C76A6DBD29E}.Release|x86.ActiveCfg = Release|x86
 		{662B587F-97B5-4CEF-ABF9-6C76A6DBD29E}.Release|x86.Build.0 = Release|x86
+		{94B50277-FB50-4B42-BA79-770ADB24CB80}.Debug|Any CPU.ActiveCfg = Debug|x86
+		{94B50277-FB50-4B42-BA79-770ADB24CB80}.Debug|x64.ActiveCfg = Debug|Any CPU
+		{94B50277-FB50-4B42-BA79-770ADB24CB80}.Debug|x64.Build.0 = Debug|Any CPU
+		{94B50277-FB50-4B42-BA79-770ADB24CB80}.Debug|x86.ActiveCfg = Debug|Any CPU
+		{94B50277-FB50-4B42-BA79-770ADB24CB80}.Debug|x86.Build.0 = Debug|Any CPU
+		{94B50277-FB50-4B42-BA79-770ADB24CB80}.Release|Any CPU.ActiveCfg = Release|x86
+		{94B50277-FB50-4B42-BA79-770ADB24CB80}.Release|x64.ActiveCfg = Release|Any CPU
+		{94B50277-FB50-4B42-BA79-770ADB24CB80}.Release|x64.Build.0 = Release|Any CPU
+		{94B50277-FB50-4B42-BA79-770ADB24CB80}.Release|x86.ActiveCfg = Release|Any CPU
+		{94B50277-FB50-4B42-BA79-770ADB24CB80}.Release|x86.Build.0 = Release|Any CPU
```

Comment:
If you want to be able to test `Samples.Kafka` in both x86 and x64 from Visual Studio, these configurations need to be fixed. I don't know if that's something we care about or we just leave it for CI to test (not using the solution file).

Right now, the solution configuration is mapping to project configuration like this:
Solution (VS) | Project
--|--
Any CPU | x86
x64 | Any CPU
x86 | Any CPU

Reply:
Weird, I can test in Rider, just fine (I have been!) Should be fixed though I guess. I'll look into it (forgot to update these when I added and removed the project to fix the guids)

Reply:
They will work in VS as they are now, but you will not be testing on the architecture you expect. The fact that your `launchSettings.json` only has the path to `win-x64` suggests you haven't tested on x86 (even if you thought you did because you selected x86 in the VS solution configuration drop-down).

Reply:
> The fact that your launchSettings.json only has the path to win-x64 suggests you haven't tested on x86

You got me 😉  Given changing configuration is a VS-hanging event, I relied on CI to do that for me 😉

Reply:
Fixed in https://github.com/DataDog/dd-trace-dotnet/pull/1444/commits/6506c6db87d0099956fe993310a8ee419c5ad625. I'm thinking I should add a Nuke target to create a new sample app with the correct configuration etc. There's currently a lot of faff required to get this right